### PR TITLE
Remove Active Supporter check for Live TV Screens

### PR DESCRIPTION
**Pull request by nZambi-**
Small change. Tests fine on my device. Active Supporter check was used only in one spot. Satisfies #2

### CODE REVIEW

File: source/LiveTvScreen.brs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 '**********************************************************
-'** createLiveTvChannelsScreen
+'** Jellfin Roku Client - createLiveTvChannelsScreen
+'** Forked from Emby
```
Comment (JustAMan) [inline source/LiveTvScreen.brs]: Do we need this comment at all?.. Git history shows all what is needed here IMHO.

File: source/appMain.brs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 '********************************************************************
-'**  Emby Roku Client - Main
+'**  Jellyfish Roku Client - Main
+'**  Forked From Emby Roku Client
```
Comment (JustAMan) [inline source/appMain.brs]: same

**Review by JustAMan — APPROVED** [review]
Looks good but I see no point in "forked from" comments.

File: source/LiveTvScreen.brs (hunk 1 above)
Comment (nZambi-) [reply source/LiveTvScreen.brs]: Fair Comment... I'll remove it if you think that's better/cleaner.

**Pull request merged**